packages/embed/src/utils/build-iframe-src.ts

```diff
@@ -37,6 +37,7 @@ const mapOptionsToQueryParams = (type: EmbedType, embedId: string, options: UrlO
     'disable-tracking': options.disableTracking || options.enableSandbox ? 'true' : undefined,
     'disable-auto-focus': options.disableAutoFocus ? 'true' : undefined,
     '__dangerous-disable-submissions': options.enableSandbox ? 'true' : undefined,
+    'share-ga-instance': options.shareGoogleAnalyticsInstance ? 'true' : undefined,
```

Comment:
let's use the same name everywhere, so it's easier to remember.
Either `shareGaInstance` + `share-ga-instance` or `shareGoogleAnalyticsInstance` + `shareGaInstance`

Reply:
the old one was called like this, might it break?

Reply:
Old v0 code will not work with v1 lib. It is acceptable to break compatibility.

Reply:
@prilutskiy this is going to break the renderer, the best solution here is to then change everything to be `share-ga-instance` not the other way around